src/azul/service/manifest_service.py

```diff
@@ -119,19 +135,23 @@ def get_manifest(self,
                      format_: ManifestFormat,
                      catalog: CatalogName,
                      filters: Filters,
-                     object_key: Optional[str] = None) -> Tuple[str, bool]:
+                     object_key: Optional[str] = None) -> Manifest:
         """
-        Returns a tuple in which the first element is pre-signed URL to a
+        Returns a Manifest object with 'location' set to a pre-signed URL to a
         manifest in the given format and of the file entities matching the given
         filter. If a suitable manifest already exists, its location will be
         returned. Otherwise, a new manifest will be generated and its location
         returned. Subsequent invocations of this method with the same arguments
         are likely to reuse that manifest, skipping the time-consuming manifest
         generation.
 
-        The second element of the returned tuple is True if an existing
+        The 'was_cached' property of the returned object is True if an existing
         manifest was reused and False if a new manifest was generated.
 
+        The 'properties' property of the returned object contains additional
```

Comment:
```suggestion
        The 'properties' attribute of the returned object contains additional
```

Reply:
But do you really want to duplicate this information in every method that returns a Manifest object?

The attributes of the Manifest object should be documented in the Manifest class.